export.py

```diff
@@ -17,7 +17,7 @@
 USER = re.compile(r'<lj user="?(.*?)"?>')
 TAGLESS_NEWLINES = re.compile('(?<!>)\n')
 NEWLINES = re.compile('(\s*\n){3,}')
-
+TIME_ZONE = 3 # GMT+03:00 for Moscow
```

Comment:
If this must be in the code, I would rather move it up, so that the user can quickly notice and find what to modify. However, a different approach should work better: using the local timezone. I believe it is rather safe to assume that the user wants to use the same timezone as the one they’re currently in.